lib/commands.py

```diff
@@ -257,17 +274,18 @@ def local_cmd(cmd, check=True, decode=True):
     # get a decoded version of the output in any case, replacing potential errors
     output_for_logs = res.stdout.decode(errors='replace').strip()
 
-    output = res.stdout
-    if decode:
-        output = output.decode()
-
     errorcode_msg = "" if res.returncode == 0 else " - Got error code: %s" % res.returncode
     command = " ".join(cmd)
     logging.debug(f"[local] {command}{errorcode_msg}{_ellide_log_lines(output_for_logs)}")
 
     if res.returncode and check:
         raise LocalCommandFailed(res.returncode, output_for_logs, command)
 
+    output: Union[bytes, str] = res.stdout
+    if decode:
+        output = output.decode()
+    if simple_output:
+        return output.strip()
     return LocalCommandResult(res.returncode, output)
```

Comment:
Making the default `simple_output=True` actually changes the default behavior of the function.  A separate commit for this change would make it easier for review.

Reply:
Done